src/markup/patterns/drizzle/partials/sidebar.hbs

```diff
@@ -0,0 +1,43 @@
+{{#*inline "nav-item"}}
+  <a class="drizzle-Nav-item" href="{{@root.baseurl}}/{{url}}">{{label}}</a>
+{{/inline}}
+
+<div class="drizzle-Layout-nav drizzle-Nav" id="nav">
+  <div class="drizzle-Nav-header">
+    <a href="/" class="drizzle-Nav-title">{{data "global.companyName"}} Design System</a>
+
+    <a class="drizzle-Nav-header-toggle" href="#nav">
+      <svg viewBox="-1 -1 12 7" width="16" height="16">
+        <title>Menu</title>
+        <polyline fill="none" stroke="currentColor" stroke-width="2" points="0 0 5 5 10 0"/>
+      </svg>
+    </a>
+  </div>
+
+  <div class="drizzle-Nav-main" id="nav-menu">
+    <ul class="drizzle-Nav-menu">
+      <li><a href="/" class="drizzle-Nav-item">Overview</a></li>
+    </ul>
+
+    <h5 class="drizzle-Nav-item drizzle-u-marginTop">Components</h5>
+    <ul class="drizzle-Nav-menu">
+      {{#each (collections "components" sortby="order")}}
+        <li>
+          {{> nav-item label=name}}
+        </li>
+      {{/each}}
+    </ul>
+
+    <h5 class="drizzle-Nav-item drizzle-u-marginTop">Documentation</h5>
+    <ul class="drizzle-Nav-menu">
+      {{#each (pages "documentation" sortby="order")}}
+        <li>
+          {{> nav-item label=data.title}}
+        </li>
+      {{/each}}
+      <li>
+        {{> nav-item label="SVG Sprite" url="/svg/symbol/sprite.symbol.html"}}
```

Comment:
Here's another thing I don't like very much. It /does/ show all available icons, which is nice. But it doesn't clearly show how to use icons within the system. Ideally it should show:

- how to add icons
- how to size icons (in src/scss/components/_icon.scss)
- how to use an `{{icon ____}}` helper to insert icons (This helper doesn't exist. But it should!)

Since this page is auto-generated, we don't have a good way to control this stuff as-is. It'd be awesome if we could output the icon list as JSON, then pull that in as data we can access in the templates.

Reply:
It would be nice to have more control over this. Did you make a card and we can treat that as an enhancement?

Reply:
Yeah. I'll make a card for this and link to this thread. 👍